2026/build-tooling/slides.md

```diff
@@ -307,7 +307,7 @@ layout: center
 ArcGIS Maps SDK for JavaScript: Fast Development and Build Tooling
 
 Demos and additional resources available at:
-[arcg.is/esri-2025-build-tooling](https://arcg.is/esri-2025-build-tooling)
+[arcg.is/esri-2026-build-tooling](https://arcg.is/esri-2026-build-tooling)
```

Comment:
we will need to ask them for new url
they also want us to use a branded qr code this year